packages/app-bridge/src/types/Themes.ts

```diff
@@ -0,0 +1,67 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { LinkSettingsDisplay, LinkSettingsIconPosition } from './Document.ts';
+
+export interface CoverPage {
+    id(): number;
+    title(language?: string): string;
+    isPublished(): boolean;
```

Comment:
there is one missing prop:
```
hiddenInNavigation(): boolean
```

the cover page, can be published but hidden in navigation.

Reply:
As agreed with @oliverschwendener we don’t need this in frontend anymore. If it is hidden, we don’t even return it from the server. This also increases the consistency across Themes.

Reply:
I disagree. You can have the Home page published and browsable, but you don't want it to show in the navigation bar. It can be that you want it only on the logo click or anywhere else, just don't want it on the navigation.